config/enrichments/currency_conversion_config.json

```diff
@@ -11,6 +10,50 @@
 			"apiKey": "{{KEY}}",
 			"baseCurrency": "USD",
 			"rateAt": "EOD_PRIOR"
```

Comment:
@istreeter What's your opinion on exposing the caching settings in parameters here similar to what we do for the API enrichment?

Reply:
I paused for ages on this question!  It makes me nervous to expose such a powerful setting in such a user-facing piece of config.  Cache size is a balance between performance vs available memory, and the person configuring this enrichment might not know all of the context for tuning that.

On balance, though, it's better to make it configurable because we never know when we'll need to tune things at run time.  So yeah we should put it in this config.

@miike what's your opinion on the `ignoreOnError` setting that [is newly added to the sql enrichment config](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.enrichments/sql_query_enrichment_config/jsonschema/1-0-1#L204-L206)?  If we added it here, it would basically mean.... if we fail to fetch a rate because of a network error, then it's OK to emit the event without the derived context.

Reply:
I like the idea of `ignoreOnError` as there are going to be circumstances where a failed enrichment event (e.g., on checkout) is going to be annoying to not have because a currency conversion couldn't be performed.

I think this does raise the question (for both enrichments) about how you "recover" data that hasn't generated a failed event but might be NULL in the data warehouse though. I don't know that we can (or want to?) change our philosophy around the immutability of these events.